Formula/clang-format.rb

```diff
@@ -4,10 +4,10 @@ class ClangFormat < Formula
   version "2016-12-09"
 
   stable do
-    url "http://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
+    url "https://llvm.org/svn/llvm-project/llvm/tags/google/stable/2016-12-09/", :using => :svn
```

Comment:
@DomT4 Yeah, I think it worked on Sierra: https://bot.brew.sh/job/Homebrew%20Core/16937/#showFailuresLink